scripts/GT++.zs

```diff
@@ -150,6 +151,13 @@ recipes.addShaped(<miscutils:blockFrameGtTumbaga> * 2, [
 [<miscutils:itemRodTumbaga>, <ore:craftingToolWrench>, <miscutils:itemRodTumbaga>],
 [<miscutils:itemRodTumbaga>, <miscutils:itemRodTumbaga>, <miscutils:itemRodTumbaga>]]);
 
+// --- Wire Factory Controller (until GT++ changes it)
+recipes.addShaped(<gregtech:gt.blockmachines:798>, [
+[<ore:plateBlueSteel>, <gregtech:gt.blockcasings:5>, <ore:plateBlueSteel>],
+[<ore:circuitElite>, <gregtech:gt.blockmachines:355>, <ore:circuitElite>],
+[<ore:plateBlueSteel>, <gregtech:gt.blockcasings:5>, <ore:plateBlueSteel>]]);
```

Comment:
way to cheap for a completly balance breaking multi

Reply:
This puts it in line with the other multis at IV, this multi has always been out of step for no real reason. The entire multi is IV except from 4 plates. Now we can look into gating all of them at a higher tier sure but for now this fixes this discrepancy.